deployments/mumbai/usdc/roots.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fxChild": "0xCf73231F28B7331BBe3124B907840A94851f9f11"
```

Comment:
This is still incomplete because we haven't deployed right?

Reply:
yep, that's correct

Reply:
- [ ] synced offline with Kevin: going to move this fxChild address to be a constant inside the mumbai deploy script (since it's strange to have a value in roots.json that is required to exist before you can deploy

Reply:
We are doing the same thing with USDC and gov for the goerli deploy, not really anything too strange about making it a root to override it... but for something that's *always* required to exist, I think this makes sense